lib/adb.js

```diff
@@ -486,12 +486,6 @@ ADB.prototype.trackDevices = function trackDevices(callback) {
  */
 function androidDetect(config, callback) {
 	(require('./android')).detect(config, null, function (results) {
-		for (var i = 0, l = results.issues.length; i < l; i++) {
```

Comment:
Does it make sense to also no longer do the check below? I'm not sure whether that would then get collated into the output on a `ti info -t android`?

https://github.com/appcelerator/node-titanium-sdk/blob/43a65aa5693a821a936e0e9df2940c711fcde13c/lib/android.js#L409-L442

Reply:
Good question. The Android build will still report a couple of those as warnings here...
https://github.com/appcelerator/titanium_mobile/blob/2923ed6b51bd70bd9160771a7c1dcfd3b7ba2837/android/cli/commands/_build.js#L1003-L1012

Reply:
Updated PR so that build won't log warnings about missing 32-bit libs on 64-bit Linux.

I left the 32-bit library detection in the code above though. I'm not actually testing this on Linux... since we don't officially support it anymore. This is a blind change that we're asking the community to double check.